docs/unified-test-documentation/dasharo-security/200-tpm-support.md

```diff
@@ -404,3 +404,374 @@ Example output:
 ```text
 SpecVersion=2.0
 ```
+
+## TPM003.001 PCRREAD Function (Ubuntu 22.04)
```

Comment:
The title of the test case should be improved. In this test case we check the correctness of the function.